### ISSUE TITLE
[FEATURE] Report error for Unmatched HTML closing tags

### ISSUE THREAD

**Issue by jayvdb** [Riverside-Healthcare/djLint#58]
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## Feature Request

This might be beyond scope of this tool ... but ... when the input has non-matching closing tags, a html error would be nice.

Here the reformatter is nicely reformatting the `<td>...</td>` but on the next line is a dangling `</td>` which doesnt have an opening tag.

```diff
@@ -40,14 +42,17 @@
                             <td>
                                 <strong>{{ foo }}</strong>
                             </td>
-                    <td>
-                        {{ bar }}                    </td>
+                            <td>{{ bar }}</td>
                         </td>
                         <td>
...
```

Another

```diff
                 <tr>
-            <th data-field="processed">{% trans 'Processed' %}</i></th>
+                    <th data-field="processed">
+                        {% trans 'Processed' %}
+                        </i>
+                    </th>
                     <th data-field="uri">{% trans 'Uri' %}</th>
```

Note I think https://github.com/thibaudcolas/curlylint does report parse error for some of these, but it doesnt have clear error messages.  They look like:
> 163:22  Parse error: expected 'p' at 163:22     parse_error

The `p` in ^ often has nothing to do with the problem, but hints that it went off-course due to a parser problem.

I am also going to try https://pypi.org/project/zpretty/ to see if it is any better / is able to find its way around the django templatism.

**Comment by jayvdb** [comment]
A rule finding problems like this can be disabled when the template is doing something impossible to parse, which shouldn't be often for projects intending to write maintainable templates.
